code/player/Player.Money.cs

```diff
@@ -5,14 +5,14 @@ namespace Cinema;
 public partial class Player
 {
     // The players current money
-    [Net] public int Money { get; set; }
+    [Net] public double Money { get; set; }
```

Comment:
Why make this a double? That's such a big and precise number, when are we going to care about money in amounts that are smaller than 0.1? Or when is a player going to have more than 1 trillion dollars?